python/pyspark/sql/types.py

```diff
@@ -1679,6 +1678,15 @@ def from_arrow_schema(arrow_schema):
          for field in arrow_schema])
 
 
+def _require_minimum_pyarrow_version():
+    """ Raise ImportError if minimum version of pyarrow is not installed
+    """
+    from distutils.version import LooseVersion
+    import pyarrow
+    if pyarrow.__version__ < LooseVersion('0.8.0'):
```

Comment:
BTW, I usually do like `LooseVersion(pyarrow.__version__) < "0.8.0"` tho.

Reply:
Just quickly checked other codes in few places I know. Let's use `LooseVersion` for both sides as @ueshin suggested to reduce possible confusion if you wouldn't mind.

Reply:
sure, no prob